docs/usage/verticles/WatchVerticle.md

```diff
@@ -0,0 +1,40 @@
+# WatchVerticle
+
+The `WatchVerticle` is a Vert.x verticle that wraps the Java WatchService API. It allows you to monitor changes to a
+specified directory, such as the creation, modification, or deletion of files or directories within it. By default, the
+`WatchVerticle` checks for changes every 500 milliseconds, but this interval can be customized by providing a different
+value as an argument to the constructor.
+
+The `WatchVerticle` provides three methods that can be overridden by subclasses to specify custom behavior in response to
+the three types of events that the WatchService API can detect: `observedCreate`, `observedModify`, and
+`observedDelete`. Subclasses can also override the start method to specify any additional initialization or setup that
```

Comment:
Maybe you can mention here at least the "parallelProcessing" or even the "handleExisting" params.